README.md

```diff
@@ -1,81 +1,71 @@
-# Linux kernel module for Thrustmaster T300RS, T248 and (experimental) TX wheels
+# Linux kernel module for Thrustmaster T300RS, T248 and TX(experimental) wheels
+> **DISCLAIMER:** The module is now ready for near-stable USE in most force feedback games, supports rangesetting as well as gain and autocentering along with most force feedback effects. While I haven't personally come across any crashes or lockups with this version, I can't promise that they won't occur under any circumstances.
 
-## Current state
-Playable. I've made some improvements to the dynamic updating of effects, and
-while still far for perfect, the experience is slowly getting better and better.
-Some drawbacks include possible effect inaccuracies in comparison with the Windows driver and in some
-games inconsistent pedal mapping. Meaning that all pedals should be detected in games, but may be mapped incorrectly.
+![GitHub last commit (branch)](https://img.shields.io/github/last-commit/Kimplul/hid-tmff2/master)
+![GitHub](https://img.shields.io/github/license/Kimplul/hid-tmff2)
+![GitHub contributors](https://img.shields.io/github/contributors/Kimplul/hid-tmff2)
```

Comment:
I'm kind of lukewarm about shields, looks cool on GitHub but they tend to clutter up reading stuff on the command line. These three aren't too bad, but maybe change `![GitHub]()` to `![GPL-3]`?

Reply:
Haha, I totally get it. I was also bit skeptical about them, but seemed cool. I didn't want to fill the README with shields. I have renamed the shields appropriately in https://github.com/Kimplul/hid-tmff2/pull/73/commits/e5a719b530418f26d957e2cae0c56a92539cb324

Reply:
![20231004_13h17m20s_grim](https://github.com/Kimplul/hid-tmff2/assets/45141270/1b16c443-37c8-427f-b6e7-d0eee2f5bba2)